src/algorithms/fardetectors/FarDetectorLinearTracking.cc

```diff
@@ -6,7 +6,15 @@
 #include <edm4eic/TrackPoint.h>
 #include <edm4eic/TrackSegmentCollection.h>
 #include <edm4eic/vector_utils.h>
+#include <edm4hep/EDM4hepVersion.h>
+#if EDM4HEP_BUILD_VERSION < EDM4HEP_VERSION(0, 99, 0)
```

Comment:
Alt
```suggestion
#if __has_include("edm4hep/TrackerHit3DCollection.h")
```

Reply:
I always assumed we are meant to use `edm4hep_VERSION` and `EDM4HEP_BUILD_VERSION` is for internal use.

Reply:
Who knows.